src/dispatchers/touchDispatcher.ts

```diff
@@ -146,6 +146,9 @@ export module Dispatchers {
      * calls all the callbacks in the provided callbackSet.
      */
     private _measureAndDispatch(event: TouchEvent, callbackSet: Utils.CallbackSet<TouchCallback>) {
+      if (!this._translator.insideSVG(event)) {
```

Comment:
Shouldn't this only apply on `touchstart`? See `Dispatcher.Mouse`'s logic when dealing with mouse events.